### ISSUE TITLE
c++ libtraci example failed 

### ISSUE THREAD

**Issue by RanBaLuDan** [eclipse/sumo#9244]
Dear all,

I tried c++ libtraci example in windows. However when I run my project in Debug mode, it shows the following error:

![image](https://user-images.githubusercontent.com/77910076/134879423-fe738315-47a3-4bb5-917e-1c84ffd571be.png)
 
I checked the net.xml file location is correct. 
All required lib, h and dll files were included in my project.
Now I have no idea, what I should do...

Your help will be appreciated.
RanBaLuDan

**Comment by namdre** [comment]
@behrisch  possibly duplicate of #9210 ?

**Comment by behrisch** [comment]
Yes, I strongly suspect that. @RanBaLuDan  Can you recheck whether it works with a nightly build? (https://sumo.dlr.de/docs/Downloads.php#nightly_snapshots)
